.cicd/scripts/srw_unittest.sh

```diff
@@ -0,0 +1,32 @@
+#!/user/bin/env bash
```

Comment:
Attempting to run the script manually is failing.  This line should be:

`#!/usr/bin/env bash`

```suggestion
#!/usr/bin/env bash
```